test/basic.test.js

```diff
@@ -509,6 +509,51 @@ test('handles objects with null prototype', async ({ same }) => {
   })
 })
 
+test('normalize bigint to string', async ({ same }) => {
+  const stream = sink()
+  const instance = pino(stream)
+  const bigInt = 1n
+  instance.info({ bigInt })
+  const result = await once(stream, 'data')
+  delete result.time
+  same(result, {
+    pid,
+    hostname,
+    level: 30,
+    bigInt: `${bigInt.toString()}n`
+  })
+})
+
+test('normalize nested object with bigint to string', async ({ same }) => {
+  const stream = sink()
+  const instance = pino(stream)
+  const bigInt = { a: { b: 1n } }
+  instance.info({ bigInt })
+  const result = await once(stream, 'data')
+  delete result.time
+  same(result, {
+    pid,
+    hostname,
+    level: 30,
+    bigInt: { a: { b: '1n' } }
+  })
+})
+
+test('object and format bigInt property', async ({ same }) => {
+  const stream = sink()
+  const instance = pino(stream)
+  instance.info({ answer: 42n }, 'foo %s', 1n)
+  const result = await once(stream, 'data')
+  delete result.time
+  same(result, {
+    pid,
+    hostname,
+    level: 30,
+    msg: 'foo 1',
```

Comment:
Why doesn't this have the `n` suffix?

Reply:
tbh idk. is it important? would you like me to add the suffix?

Reply:
The other outputs in this PR have the suffix. I'm asking about it from a consistency standpoint. How will an interpreter know that it is a `BigInt` without the suffix?